Demo/Application/Features/3D Secure/BraintreeDemoThreeDSecurePaymentFlowViewController.m

```diff
@@ -192,6 +192,18 @@ - (void)tappedToVerifyNewCard {
 
         request.v2UICustomization = ui;
 
+        // MARK: - UI and Render Type Customization
+
+        request.uiType = BTThreeDSecureUITypeBoth;
+        request.renderTypes = [[NSArray alloc] initWithObjects:
+                              BTThreeDSecureRenderTypeHTML,
+                              BTThreeDSecureRenderTypeOOB,
+                              BTThreeDSecureRenderTypeOTP,
+                              BTThreeDSecureRenderTypeMultiSelect,
+                              BTThreeDSecureRenderTypeSingleSelect,
+                              nil
+        ];
+
```

Comment:
Jax, can you give me quick explanation of what these renderTypes are for? What does Cardinal do with it?

Reply:
These values were recommended to a merchant to resolve an issue with certain banks. Live issue LI-8308 has more details if you're interested. The [Cardinal docs](https://cardinaldocs.atlassian.net/wiki/spaces/CMSDK/pages/2005696619/Setting+up+CardinalMobileSDK+-+iOS#Available-Configurations--%5BinlineExtension%5D) don't really have much detail but is all we have to go on. Their docs indicate that the UI and render type are used to specify the challenge type supported by the device.